projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -115,18 +99,16 @@ const NewsletterDisabledPanels = () => (
 	</>
 );
 
-function NewsletterPrePublishSettingsPanel( {
-	accessLevel,
-	setPostMeta,
-	socialFollowers,
-	emailSubscribers,
-	paidSubscribers,
-	isModuleActive,
-	showMisconfigurationWarning,
-	showPreviewModal,
-} ) {
+function NewsletterPrePublishSettingsPanel( { setPostMeta, isModuleActive, showPreviewModal } ) {
 	const { tracks } = useAnalytics();
 	const { changeStatus, isLoadingModules, isChangingStatus } = useModuleStatus( name );
+	const { accessLevel } = useSelect( select => {
```

Comment:
Wouldn't writing it this way be a bit more readable? 

```suggestion
	const accessLevel = useSelect( select =>  select( 'jetpack/membership-products').getAccessLevel() );
```

Instead of wrapping it in an object to extract it straight away?

Reply:
Yes, this is much more readable. I'll make these changes, thanks!

Reply:
[Object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) is pretty useful and powerful in JS